code/modules/surgery/implant.dm

```diff
@@ -175,6 +175,12 @@
 		to_chat(user, "<span class='warning'>Central command would kill you if you implanted the disk into someone.</span>")
 		return 0//fail
 
+	if(istype(tool, /obj/item/weapon/storage/box))
```

Comment:
It's still possible to use fannypacks or wallets with this. I'd change it to just `/obj/item/weapon/storage`.